### ISSUE TITLE
FR: Accessibility revision (edition mode)

### ISSUE THREAD

**Issue by ignaciogros** [exelearning/iteexe#294]
We revised the exported contents, but we still have to check the accessibility issues while editing. For example: After adding an iDevice, the instructions are at the end of if, and they should be at the beginning (at least a link to those instructions), the controls should be properly labeled, etc.
